### ISSUE TITLE
Fix JavaDoc integration in module docs

### ISSUE THREAD

**Issue by jdrueckert** [Terasology/Health#101]
The API sections in the module docs are currently leading to
![image](https://user-images.githubusercontent.com/29981695/233804369-cc6a1ec6-9093-4fb3-a9b1-fdd3648b05be.png)

The links in question are
- http://jenkins.terasology.io/teraorg/job/Terasology/job/Modules/job/H/job/Health/job/master/javadoc/overview-summary.html
- http://jenkins.terasology.io/teraorg/job/Terasology/job/Modules/job/H/job/Health/job/develop/javadoc/overview-summary.html

This is due to the switch to a newer jenkins (http://newerjenkins.terasology.io instead of https://jenkins.terasology.io).

The new javadocs will be available on
- http://newerjenkins.terasology.io/job/Terasology/job/Modules/job/H/job/Health/job/develop/...
- http://newerjenkins.terasology.io/job/Terasology/job/Modules/job/H/job/Health/job/master/...

once they ran for the first time which they did not do yet.

Blocks #104

**Comment by Cervator** [comment]
Is this a limited issue on just one or a few modules needing it? I expect the URL will change again, maybe even during the fun a week from now. Easy enough to change a bit, especially if new URL happens later, just wouldn't want to change something per-module while on a temp setup :-)

**Comment by jdrueckert** [comment]
> Is this a limited issue on just one or a few modules needing it? I expect the URL will change again, maybe even during the fun a week from now. Easy enough to change a bit, especially if new URL happens later, just wouldn't want to change something per-module while on a temp setup :-)

Fair point. I don't think we have this kind of doc structure in a lot of modules yet, so it's more of a one off fix. But if you expect it to change it soon again anyway, we can set it to blocked for now. Just have to remember to eventually unblock it again.
